timesketch/__init__.py

```diff
@@ -70,6 +70,10 @@ def create_app(config=None):
         os.environ['TIMESKETCH_SETTINGS'] = config
         try:
             app.config.from_envvar('TIMESKETCH_SETTINGS')
+            if app.config.get('EMAIL_RECIPIENTS') is None:
```

Comment:
None is default if not found. 

```suggestion
            if not app.config.get('EMAIL_RECIPIENTS'):
```